supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

```diff
@@ -46,7 +46,7 @@
 
 [[pc]]
 ==== image:images/yes.png[yes] PC (noun)
-*Description*: _PC_ is an abbreviation for personal computer. For more information, see the _IBM Style_ guide.
+*Description*: _PC_ is an abbreviation for personal computer. For more information, see the _microcomputer_ entry in the *Word Usage* section of the _IBM Style_ guide.
```

Comment:
I think the ISG reference doesn't add any value. I'd just delete the second sentence.

Reply:
Thanks, @bredamc . So, drop the reference to the IBM SG?

Reply:
Yes, that's my preference, but see what other reviewers think :)

Reply:
Agree with Breda that we should drop the second sentence. The "microcomputer" entry isn't helpful in this case.

Reply:
Thanks, @bredamc , @bergerhoffer , and @mportman12 . Updated the content and the preview link.